Achtung! Cthulhu CoC7 - Release v1.03

zedafty opened the pull request:
<!-- ATTENTION: This Pull Request template changed on 03/17/22. Please ensure that you are completing this template to the fullest extent possible. -->

# Submission Checklist
## Required

<!-- Check these off by adding an 'x' to each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

# Changes / Description

- Fix extreme damage formula (maximum weapon damage + maximum damage bonus + impalement)
- Fix chatlog action button call based upon character name instead of character id
- Fix occupation and interest skill points totals update on skill points reset
- Fix monster attacks extreme damages and damage bonus
- Fix Show Scores parameter not checked by default
- Replace attribute "imp" by "dmg-max"
- Add Call of Cthulhu dark visual theme
- Add printable sheet option with print friendly styles
- Add persistant insanity cap number bubble box and recalc button
- Add major wound cap number bubble box
- Add description roll template (for out of sheet macros)
- Add empty rows sheet option for repeating sections on sheet creation
- Add no damage chatlog text (instead of nothing displayed)
- Add limits to minimal and maximal Credit Rating
- Merge Investigator and Achtung! PC sheet types into PC sheet type
- Remove advancement checkbox for Credit Rating and Cthulhu Mythos
- Reset inline rolls default styles (e.g. use roll result in roll name)
- Show fighting skill score next to weapon name
- Show options blocks expanded by default
- Make Cash field editable in view mode and auto-computed (i.e. calculate sums like 1+2+3 up to four operands)
- Set maximum characteristic score for players to 99 (including Luck, saving Size [200 max] and Power [150 max])
- Set maximum skill score for players to 109 (99 + 1D10 due to advancement ; except for Credit Rating and Cthulhu Mythos)

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/achtung-cthulhu-coc7/roll20-character-sheets) *Roll20 Internal Use only.*

nmbradley commented:
Hi @zedafty 

Thanks for the pull request. The updated sheet looks incredible.

We have to clear the usage of logos and so forth with our publishing partners and we've submitted some requests via our Partner Relationships team to ensure that they are comfortable with their logos being used in this way.

We should expect a response in a few days, and we'll let you know if any further steps are required.

kfroll20 commented:
Hello @zedafty! 

We heard back from our Partner Relations team and unfortunately we cannot merge this PR until the logos are removed since this sheet is not official. Once those logos are removed, we can go forward with getting this merged. Thank you for your understanding.

zedafty commented:
Thank you for the feedback and explanations @nmbradley and @kfroll20 !

I understand the matter. Does a custom logo, not using the Chaosium logo font, would be ok for the character sheet ?

kfroll20 commented:
> Thank you for the feedback and explanations @nmbradley and @kfroll20 !
> 
> I understand the matter. Does a custom logo, not using the Chaosium logo font, would be ok for the character sheet ?

As long as you're not using the logo, font, or either that are indistinguishable from the official one, you should be good to go. We appreciate your understanding :)

zedafty commented:
Thank you for response @kfroll20 !

I removed the official logos for Call of Cthulhu theme and replaced them with unofficial ones using a typewriter font that I hope should be different enough to prevent any licenses issue :)

NorWhal commented:
Hey @zedafty! 

This all looks great, but we do need the preview to be updated too. Tag me when you've uploaded the new preview image and I can do my best to get this merged ASAP. Thank you so much for your understanding. 

Nora
